### PR TITLE
Add last_seen_at timestamp in system object to docs

### PR DESCRIPTION
**Pull request by Sergeykot**

### CODE REVIEW

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/127203/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged baed592307b3a8f9718d942d6192dc4d9475b188 into 7f743bc184a87b995ea2f02008cdb3b22387b839)

File: doc/SCC-API-(Implemented).md [hunk 1]
```diff
@@ -458,7 +460,7 @@ Status: 200 OK
     "product_classes": ["SLES"],
     "families": ["sles", "sled"],
     "skus": ["sku1", "sku2"],
-    "systems": [{ "id": 1, "login": "login1" }]
+    "systems": [{ "id": 1, "login": "login1", "last_seen_at" : "2010-03-14T13:10:21.164Z" }]
```
Comment (wstephenson) [inline doc/SCC-API-(Implemented).md]: Extra space between key name and colon

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/127413/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged e6acc0e8337ff2ed60cf883861a12dcbfd177b45 into 7f743bc184a87b995ea2f02008cdb3b22387b839)

**Comment by Sergeykot** [comment]
needs another review (added script example to the PR)

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/SCC-Connect-Prophet/127766/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests-sle12sp3. (Merged 96760b29608258c3bf06ad5fb5816c022b4ae394 into 7f743bc184a87b995ea2f02008cdb3b22387b839)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/127786/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 94c111a7defd5d3b1f949111fd6ed4270953c733 into 7f743bc184a87b995ea2f02008cdb3b22387b839)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/131739/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged 94c111a7defd5d3b1f949111fd6ed4270953c733 into f1091c5316af298b3e9d427a2608e9f83b082b93)

File: examples/delete_systems.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/ruby
+
+require 'rest-client'
+require 'net/http'
+require 'pp'
+require 'json'
+require 'base64'
+
+require 'byebug'
```
Comment (felixsch) [inline examples/delete_systems.rb]: I think no debugging tools are required here

File: examples/delete_systems.rb [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        API_BASE_URL + '/systems',
+        {
+          Authorization: 'Basic ' + Base64.encode64("#{system['login']}:#{system['password']}").delete("\n")
+        }
+      )
+    end
+  end
+end
+
+resp = get(API_BASE_URL + 'organizations/systems')
```
Comment (felixsch) [inline examples/delete_systems.rb]: Why not replace `resp` with `links`?
```
links = { next: API_BASE_URL + 'organizations/systems' }
while links[:next] do
  resp = get(links[:next])
  process_data(resp)
  links = process_rels(resp)
end
```
Looks more clean to me

File: examples/delete_systems.rb [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+@page = 1
```
Comment (felixsch) [inline examples/delete_systems.rb]: What is the tought behind the `@page` variable? seems like its not used anywhere.

**Comment by suse-tests-fail** [comment]
Prophet reports failure.
https://ci.suse.de/job/SCC-Connect-Prophet/132027/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment.
Failed fast in project connect on step docker-run-tests-sle12sp3. (Merged 64914e58450089ab1fe38d14fb1aa002cf648b28 into f1091c5316af298b3e9d427a2608e9f83b082b93)

**Comment by suse-tests-pass** [comment]
Well Done! Your tests are still passing.
https://ci.suse.de/job/SCC-Connect-Prophet/132029/console
If the given link has expired,you can force a Prophet rerun by just deleting this comment. (Merged b5062e181e341e7eb8f4b15b5f108dd13cf51fe3 into f1091c5316af298b3e9d427a2608e9f83b082b93)

**Pull request merged**